sdk/turing/session.py

```diff
@@ -273,6 +273,14 @@ def list_router_versions(self, router_id: int) -> List[RouterVersion]:
             router_id=router_id
         )
 
+    @require_active_project
```

Comment:
Hmm I guess it doesn't hurt to have one, but if we're doing it, shouldn't it be an instance method of `Router` because we could simply use an `Router` instance's `id` attribute to call the  `create_router_version` method you've quoted, or simply the `RouterVersion.create` class method, both of which require a `router_id` as an argument.

Reply:
Apologies. I did mean to say "instance method".